Lk loom schema updates (#158)
broadinstitute/warp: website/docs/documentation/Pipelines/Optimus_Pipeline/Loom_schema.md

|`molecules_with_single_read_evidence`|[SC Tools](https://github.com/HumanCellAtlas/sctools/tree/master/src/sctools/metrics)| The number of molecules associated with this entity that are observed by only one read. [Metrics Definitions](https://sctools.readthedocs.io/en/latest/sctools.metrics.html#sctools.metrics.aggregator.CellMetrics.molecules_with_single_read_evidence)|
|`number_cells_detected_multiple`|[SC Tools](https://github.com/HumanCellAtlas/sctools/tree/master/src/sctools/metrics)| The number of cells which observe more than one read of this gene. [Metrics Definitions](https://sctools.readthedocs.io/en/latest/sctools.metrics.html#sctools.metrics.aggregator.GeneMetrics.number_cells_detected_multiple)|
|`number_cells_expressing`|[SC Tools](https://github.com/HumanCellAtlas/sctools/tree/master/src/sctools/metrics)| The number of cells that detect this gene. [Metrics Definitions](https://sctools.readthedocs.io/en/latest/sctools.metrics.html#sctools.metrics.aggregator.GeneMetrics.number_cells_expressing)|

## HCA Data Coordination Platform Matrix Processing

> Optimus was designed for processing data from the Human Cell Atlas (HCA) Data Coordination Platform (DCP; see the DCP [Data Portal here](https://data.humancellatlas.org/)). 

Reviewer: ```suggestion
Optimus supports data processing for the Human Cell Atlas (HCA) Data Coordination Platform (DCP; see the DCP [Data Portal here](https://data.humancellatlas.org/)). 
```

It was originally developed for HCA but I don't want to imply that's all it's good for.

**These matrices are minimally filtered so that only cells with 100 molecules or more are retained.**

:::warning
This section details matrices produced for the Human Cell Atlas (HCA) [Data Coordination Platform (DCP)2.0](https://data.humancellatlas.org/), which includes matrices processed with Optimus v4.1.7 and later. The DCP is currently reprocessing data generated with earlier Optimus versions and will deprecate previous matrices once reprocessing is complete. 
:::

HCA DCP matrices contain HCA metadata in the Loom global attributes (see table below) which may be useful when exploring the data and linking it back to the Project metadata. Read more about each metadata field in the HCA DCP [Metadata Dictionary](https://data.humancellatlas.org/metadata). 

| Metadata Attribute Name in Count Matrix | Metadata Description | 
| --- | --- |
| `donor_organism.genus_species` | species information; human or mouse |
| `library_preparation_protocol.library_construction_method` | technology used for library preparation, i.e 10x or SS2 |
| `specimen_from_organism.organ` | organ |	
| `project.project_core.project_name` | project name |
| `project.provenance.document_id` | project id |

> | `input_id` | metadata values for  `sequencing_process.provenance.document_id`; unique ID to demarcate the library prep |

Reviewer 2: We also have input_id_metadata_field and input_name_metadata_field

Author: @khajoue2 Are those being carried over into the HCA Project matrices?

| `input_name` | metadata values for `sequencing_input.biomaterial_core.biomaterial_id`; unique ID for the biomaterial |

The `input_id` is a comma-separated string containing the IDs for the library prep from which a cell came and can be mapped back to the unique cell barcodes that are listed in the Loom `cell_names` column attribute. 

> Each barcode in the `cell_names` attribute has a numerical suffix (i.e. "-1", "-2", etc.); this suffix is an index for the input IDs. For example, barcodes with "-1" comprise the first ID listed in the `input_id` string, whereas barcodes with a "-2" comprise the second ID listed in the `input_id` string.

Reviewer 3: these are currently 0 indexed
